cadc-access-control-server/src/main/java/ca/nrc/cadc/ac/server/ldap/LdapConfig.java

```diff
@@ -115,7 +116,8 @@
     public enum PoolPolicy
     {
         roundRobin,
-        fewestConnections
+        fewestConnections,
+        firstResponse
```

Comment:
This must be renamed to fastestConnect (and in the example config) to relfect what it actually does.